<ui version="4.0" >
 <class>SelectionPage</class>
 <widget class="QWizardPage" name="SelectionPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Parent gallery</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Parent gallery.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="parentBox" >
        <item>
         <property name="text" >
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>General</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Please select a name for this new gallery.</string>
        </property>
        <property name="buddy" >
         <cstring>nameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="GWidgets::LineEdit" name="nameEdit" />
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Please select the folder that contains the pictures you'd like to add your gallery.</string>
        </property>
        <property name="buddy" >
         <cstring>imagesEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2" >
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>6</number>
        </property>
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <widget class="GWidgets::LineEdit" name="imagesEdit" />
        </item>
        <item>
         <widget class="QPushButton" name="browseButton" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../images.qrc" >:/images/folder-open.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Preview</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QListWidget" name="previewList" >
        <property name="editTriggers" >
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode" >
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="iconSize" >
         <size>
          <width>128</width>
          <height>128</height>
         </size>
        </property>
        <property name="isWrapping" stdset="0" >
         <bool>true</bool>
        </property>
        <property name="viewMode" >
         <enum>QListView::IconMode</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GWidgets::LineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>QWizardPage</class>
   <extends>QWidget</extends>
   <header location="global" >QtGui/QWizard</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameEdit</tabstop>
  <tabstop>imagesEdit</tabstop>
  <tabstop>browseButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../images.qrc" />
 </resources>
 <connections/>
</ui>
